packages/fx-core/resource/package.nls.json

```diff
@@ -901,6 +901,7 @@
   "driver.botAadApp.log.failExecuteDriver": "Unable to execute action %s. Error message: %s",
   "driver.botAadApp.log.successCreateBotAad": "Created Azure Active Directory application with client id %s.",
   "driver.botAadApp.log.useExistingBotAad": "Used existing Azure Active Directory application with client id %s.",
+  "driver.botAadApp.error.unexpectedEmptyBotPassword": "Unexpected empty bot password. You can choose to add your own generated bot password in env file or clear bot id as well so that a new pair would be generated. action: %s.",
```

Comment:
@sffamily  @MuyangAmigo Hi can you help review this error message in terms of wording? thanks.

Reply:
Try to make it simple and straightforward:

Bot password is empty. Add it in env file or clear bot id to have bot id/password pair regenerated. action: %s.

Reply:
Though users can find corresponding envs in `writeToEnvironmentFile` field, do we need to tell users which env is empty / needs to be filled?

Reply:
yes, if it can cause confusion, we need to be very specific, may need to specify the env file name.